scss/typography/_dropcaps.scss

```diff
@@ -2,13 +2,30 @@
 /// @group typography-dropcaps
 ////
 
+/// let 1em = 16px for the conversion tool
+/// @type Number
+$dropcaps-browser-context: 16;
+
+/// Safe px to em conversion
+@function em($pixels, $context: $dropcaps-browser-context) {
+  @if (unitless($pixels)) {
+    $pixels: $pixels * 1px;
+  }
+
+  @if (unitless($context)) {
+    $context: $context * 1px;
+  }
+
+  @return $pixels / $context * 1em;
+}
+
 /// Default number of lines against the rest of the content  
 /// @type Number
 $dropcaps-lines: 3;
 
 /// Default line height for the rest of the content
 /// @type Number
-$dropcaps-line-height: 1.6;
+$dropcaps-line-height: 1.5;
```

Comment:
`1.6` by default, like paragraphs (or maybe you can use `$paragraph-line-height`)